bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/RetryingRpcFutureFallback.java

```diff
@@ -80,7 +80,6 @@ private RetryingRpcFutureFallback(RetryOptions retryOptions, RequestT request,
     }
     long nextBackOff = currentBackoff.nextBackOffMillis();
     if (nextBackOff == BackOff.STOP) {
-      LOG.warn("RetriesExhausted: ", cause);
       throw new ScanRetriesExhaustedException("Exhausted streaming retries.", cause);
```

Comment:
NOTE: The underlying cause of the retries exhausted is included in the ScanRetriesExhaustedException.  It should not be logged here, since the user can figure out how to handle the exception, including logging.